base/src/main/java/org/bstats/charts/SimplePie.java

```diff
@@ -19,6 +20,16 @@ public SimplePie(String chartId, Callable<String> callable) {
         this.callable = callable;
     }
 
+    /**
+     * Class constructor.
+     *
+     * @param chartId The id of the chart.
+     * @param callable The callable which is used to request the chart data
+     */
+    public SimplePie(String chartId, BooleanCallable callable) {
+        this(chartId, (Callable<String>) callable);
```

Comment:
```suggestion
    public SimplePie(String chartId, Callable<Boolean> callable) {
        this(chartId, () -> String.valueOf(callable.call()));
```
You can get rid of the `BooleanCallable` by just using a `Callable<Boolean>`. Autoboxing takes care that it also works with the primitive `boolean` type.
This can

Reply:
Callable<Boolean> is not possible here because of type erasure. Causes a compile error. Thus the reason for the BooleanCallable interface.

Reply:
Oops... Yes, makes sense.